src/next-transpile-modules.js

```diff
@@ -99,13 +99,13 @@ const withTmInitializer = (transpileModules = []) => {
           );
 
           if (nextCssLoader) {
-            nextCssLoader.issuer.include = nextCssLoader.issuer.include.concat(includes);
-            nextCssLoader.issuer.exclude = excludes;
+            nextCssLoader.issuer.and = nextCssLoader.issuer.and.concat(includes);
+            nextCssLoader.issuer.not = excludes;
           }
 
           if (nextSassLoader) {
-            nextSassLoader.issuer.include = nextCssLoader.issuer.include.concat(includes);
-            nextSassLoader.issuer.exclude = excludes;
+            nextSassLoader.issuer.and = nextSassLoader.issuer.and.concat(includes);
```

Comment:
:o I'm surprised `nextCssLoader` being used instead of `nextSassLoader ` wasn't causing issues before...

Reply:
🙈 apparently it worked fine

Reply:
luckily they had the same includes that's why it didn't cause any problems 😄